tests/test_baseline.py

```diff
@@ -116,3 +116,36 @@ def testBaselineRegenerationNeeded(baselineFile, tmpFile: Path):
 
     assert baselineRegenerationNeeded is True
     assert clearedViolations == {}
+
+
+def test_baseline_indent(
+        tmp_path: Path, monkeypatch: pytest.MonkeyPatch
+) -> None:
+    """Confirm round trip equality with a space or tab indent in the baseline file.
+
+    Args:
+        tmp_path: Temporary path.
+        monkeypatch: Pytest monkeypatch fixture
+    """
+
+    code_file = tmp_path / 'code.py'
+    baseline_spaces = tmp_path / 'baseline_spaces.txt'
+    baseline_tabs = tmp_path / 'baseline_tabs.txt'
+
+    code_file.write_text(badDocstringFunction)
+    violations = _checkPaths((str(code_file),), exclude=EXCLUDE_PATTERN)
+
+    generateBaseline(violations, baseline_spaces)
+
+    monkeypatch.setattr('pydoclint.baseline.INDENT', '\t')
+    generateBaseline(violations, baseline_tabs)
+
+    assert baseline_spaces.read_text().splitlines()[1].startswith('    ')
+    assert baseline_tabs.read_text().splitlines()[1].startswith('\t')
+
+    key = str(code_file)
```

Comment:
> @jsh9 Any hints here? I don't have easy access to a windows box to sort the path issue.
> 
> Any guidence appreciated. - brad

Hi @cidrblock ,

I also don't have a Windows machine at this moment, but maybe we can infer something from the error message:

```
================================== FAILURES ===================================
  ____________________________ test_baseline_indent _____________________________
  
  tmp_path = WindowsPath('C:/Users/runneradmin/AppData/Local/Temp/pytest-of-unknown/pytest-0/test_baseline_indent0')
  monkeypatch = <_pytest.monkeypatch.MonkeyPatch object at 0x0000029920DA8E80>
  
      def test_baseline_indent(
              tmp_path: Path, monkeypatch: pytest.MonkeyPatch
      ) -> None:
          """Confirm round trip equality with a space or tab indent in the baseline file.
      
          Args:
              tmp_path: Temporary path.
              monkeypatch: Pytest monkeypatch fixture
          """
      
          code_file = tmp_path / 'code.py'
          baseline_spaces = tmp_path / 'baseline_spaces.txt'
          baseline_tabs = tmp_path / 'baseline_tabs.txt'
      
          code_file.write_text(badDocstringFunction)
          violations = _checkPaths((str(code_file),), exclude=EXCLUDE_PATTERN)
      
          generateBaseline(violations, baseline_spaces)
      
          monkeypatch.setattr('pydoclint.baseline.INDENT', '\t')
          generateBaseline(violations, baseline_tabs)
      
          assert baseline_spaces.read_text().splitlines()[1].startswith('    ')
          assert baseline_tabs.read_text().splitlines()[1].startswith('\t')
      
          key = str(code_file)
  >       space_parsed = sorted(parseBaseline(baseline_spaces)[key])
  E       KeyError: 'C:\\Users\\runneradmin\\AppData\\Local\\Temp\\pytest-of-unknown\\pytest-0\\test_baseline_indent0\\code.py'
  
  tests\test_baseline.py:1[47](https://github.com/jsh9/pydoclint/actions/runs/9908693585/job/27542366985?pr=152#step:5:48): KeyError
```

So it seems like Windows adds an additional `\` to the file path (making them `\\`).

So maybe the solution is: instead of doing `str(code_file)`, you can try `code_file.as_posix()`: https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.as_posix

Reply:
Excellent, TY, appreciate the hint